conan/internal/model/conan_file.py

```diff
@@ -386,8 +387,11 @@ def run(self, command: str, stdout=None, cwd=None, ignore_errors=False, env="",
         wrapped_cmd = command_env_wrapper(self, command, env, envfiles_folder=envfiles_folder)
         from conan.internal.util.runners import conan_run
         if not quiet:
-            ConanOutput().writeln(f"{self.display_name}: RUN: {command}", fg=Color.BRIGHT_BLUE)
+            ConanOutput().info(f"{self.display_name}: RUN: {command}", fg=Color.BRIGHT_BLUE)
         ConanOutput().debug(f"{self.display_name}: Full command: {wrapped_cmd}")
+        if quiet or ConanOutput()._conan_output_level == LEVEL_QUIET:
```

Comment:
I don't love the fact that changes to how Conan outputs things are being done for the new ``conan run`` experimental functionality. I think these changes are riskier than they appear, and should be avoided, or at least considered more carefully.